easybuild/easyconfigs/f/FFTW/FFTW-2.1.5_fix-configure-Fortran-linking.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/fftw-2.1.5.orig/configure b/fftw-2.1.5/configure
```

Comment:
Ah, it was there before, but it got wiped because I screwed something up... Thanks for the reminder!